NuclearData/ceNeutronData/ceNeutronNuclide_inter.f90

```diff
@@ -262,6 +295,45 @@ pure function isFissile(self) result(isIt)
 
   end function isFissile
 
+  !!
+  !! Set the hasDBRC flag to .true. if called
+  !!
+  !! Args:
+  !!   None
+  !!
+  !! Result:
+  !!   None
+  !!
+  !! Errors:
+  !!   None
+  !!
+  subroutine setDBRC(self)
+    class(ceNeutronNuclide), intent(inout) :: self
+
+    self % DBRC = .true.
+
+  end subroutine setDBRC
```

Comment:
Perhaps we should move it as an additional keyword in the `set` procedure. 
The original intention of having a single procedure that would be called with keyword, optional arguments was to simply the interface and avoid having to write may `set` procedures. I don't know if it ends up being convenient in practice though...

Reply:
I think it's not convenient with how things are implemented now, because setDBRC is called from aceNEutronDatabase and not in aceNeutronNuclide. So I would need to modify things a lot and maybe make them less neat (now we have a function called initDBRC in the aceNeutronDB, if I try to use set I would need to modify init and probably make it a bit messier). I don't think it's worth it.

Reply:
I don't know if it would really be much messier. I mean it would just boil down to changing the line 760 of `aceNeutronDatabase_class.f90` from:
```Fortran
call self % nuclides(nucSet % atVal(j)) % setDBRC()
```
to e.g.: 
```Fortran
call self % nuclides(nucSet % atVal(j)) % set(dbrc=.true.)
```
Basically there is no problem calling `set` multiple times in different places since if the keyword is not provided on invocation, the associated entry is simply unchanged.   
There is a good chance I am missing something but I have a trouble to understand what would necessitate a rewrite of the init procedures.

Reply:
Sure, I see! I think I hadn't realised that all the arguments of set were optional and one could just call it like that. Very convenient